src/controllers/bookController.ts

```diff
@@ -25,6 +28,39 @@ class BookController {
             error_description: 'Endpoint not implemented yet.',
         });
     }
+
+    getAllBooks(req: Request, res: Response) {
+        const allBooks: Book[] = [];
+        const connection = new TediousConnection(dbConfig);
+
+        connection.on('connect', function (err: Error | undefined) {
+            if (err) {
+                res.json(JSON.stringify(err));
+                return;
+            }
+            const request = new TediousRequest(
+                'select * from bookish.dbo.book',
```

Comment:
I have a slight preference for the following SQL style:
- Identifiers in `lowercase` and `snake_case`
- Keywords and clauses in `UPPERCASE`

Reply:
As long as you distinguish your identifiers and keywords, that's the important bit